services/slam/client_test.go

```diff
@@ -70,10 +78,37 @@ func TestClientWorkingService(t *testing.T) {
 		return mimeType, imSucc, nil, nil
 	}
 
+	workingSLAMService.GetPointCloudMapStreamFunc = func(ctx context.Context, name string) (func() ([]byte, error), error) {
+		reader := bytes.NewReader(pcd)
+		clientBuffer := make([]byte, chunkSizePointCloud)
+		f := func() ([]byte, error) {
```

Comment:
Note: We considered separating this out into a separate function that takes in a buffer and size but chose to leave it explicit

Reply:
this is good